### DISCUSSION TITLE
problems with the last update

### DISCUSSION THREAD

**Discussion by ejsnews** [oobabooga/text-generation-webui#3966] [General]
Hello,
I did an installation with start_windows.bat the 14 Aug 2023 and that was running quite well with many models GGML, GPTQ, GGUF but today the 17 Sept 2023 i did the error to use update_windows.bat for have an update.
I did not seen any problem during this update, but the http://127.0.0.1:7860/ is now giving a new interface where GGML are dead to load and other models are crazy and unable to use. The chat interface seems have lost many options, it is now impossible to use this oobabooga.
Is exist a way to come back to the last version or use another application to use my models fully on windows ?

Thanks

**Comment by missionfloyd** [comment]
GGML support was [removed](https://github.com/oobabooga/text-generation-webui/commit/ed86878f02b6b6194450627d85e8b38f84d3f960). Replace your GGML models with their GGUF versions (where available), or revert with `git reset --hard cc7b7ba153874cd3b5d6e2ab6fed23ade2dc7782`

> The chat interface seems have lost many options, it is now impossible to use this oobabooga.

By default, all the options are in the ☰ button to the left. To use the old interface, enable `--chat-buttons` on the Session tab (or put it in `CMD_FLAGS.txt`).
